### DISCUSSION TITLE
disable healthcheck on build

### DISCUSSION THREAD

**Discussion by mluhmann** [containers/podman#16643] [Q&A] [answered]
Hello,

I have a custom linux build on yocto without systemd and I had to update to podman v4.2.1. But now I am getting lots of errors due to missing systemd like "unable to get systemd connection to add healthchecks". I can work around this with the --no-healthcheck option or disable it in my compose files. But this isn't convenient for legacy containers and developers working with this linux. 

Is there an option to disable healthcheck for my whole build by default?

**Comment by vrothberg** [comment, marked as answer]
Thanks for reaching out, @mluhmann!

> Is there an option to disable healthcheck for my whole build by default?

No, there's no global config option for that.

Would you open an issue?  Something seems off.

**Reply by mluhmann** [reply]
https://github.com/containers/podman/issues/16644
